scripts/echo_services_markdown.py

```diff
@@ -0,0 +1,127 @@
+#!/bin/env python
+""" Usage
+
+    cd osparc-simcore
+    ./scripts/echo_services_markdown.py >services.md
+"""
+
+import itertools
+import sys
+from collections.abc import Iterable
+from datetime import datetime
+from pathlib import Path
+from typing import Final, NamedTuple
+
+CURRENT_FILE = Path(sys.argv[0] if __name__ == "__main__" else __file__).resolve()
+CURRENT_DIR = CURRENT_FILE.parent
+
+_URL_PREFIX: Final[
+    str
+] = "https://raw.githubusercontent.com/ITISFoundation/osparc-simcore/refs/heads/master"
+
+_REDOC_URL_PREFIX: Final[str] = f"https://redocly.github.io/redoc/?url={_URL_PREFIX}"
+_SWAGGER_URL_PREFIX: Final[str] = f"https://petstore.swagger.io/?url={_URL_PREFIX}"
+
+
+def _to_row(values: Iterable):
+    return f"| {'|'.join(map(str, values))} |\n"
+
+
+class CaptureTuple(NamedTuple):
+    service_name: str
+    file_path: Path
+
+
+def _by_service_name(e: CaptureTuple):
+    return e.service_name
```

Comment:
personally I would use a `lambda` for these small one-line utils, but I guess that's a matter of taste.